[CI][E2E] Some Matrix tests fail to build on windows build-e2e ci machine due to unknown member `high_resolution_clock`

### Describe the bug

The following tests fail due to `error: no member named 'high_resolution_clock' in namespace 'std::chrono'` on Windows CI build machines:
```
  SYCL :: Matrix/SG32/joint_matrix_bf16_fill_k_cache.cpp
  SYCL :: Matrix/SG32/joint_matrix_bf16_fill_k_cache_SLM.cpp
  SYCL :: Matrix/SG32/joint_matrix_bf16_fill_k_cache_init.cpp
  SYCL :: Matrix/SG32/joint_matrix_bf16_fill_k_cache_unroll.cpp
  SYCL :: Matrix/SG32/joint_matrix_bf16_fill_k_cache_unroll_init.cpp
  SYCL :: Matrix/joint_matrix_bf16_fill_k_cache.cpp
  SYCL :: Matrix/joint_matrix_bf16_fill_k_cache_OOB.cpp
  SYCL :: Matrix/joint_matrix_bf16_fill_k_cache_SLM.cpp
  SYCL :: Matrix/joint_matrix_bf16_fill_k_cache_arg_dim.cpp
  SYCL :: Matrix/joint_matrix_bf16_fill_k_cache_init.cpp
  SYCL :: Matrix/joint_matrix_bf16_fill_k_cache_prefetch.cpp
  SYCL :: Matrix/joint_matrix_bf16_fill_k_cache_runtime_dim.cpp
  SYCL :: Matrix/joint_matrix_bf16_fill_k_cache_unroll.cpp
  SYCL :: Matrix/joint_matrix_bf16_fill_k_cache_unroll_init.cpp
```
Example error message:
```
********************
FAIL: SYCL :: Matrix/SG32/joint_matrix_bf16_fill_k_cache_init.cpp (1591 of 2303)
******************** TEST 'SYCL :: Matrix/SG32/joint_matrix_bf16_fill_k_cache_init.cpp' FAILED ********************
Exit Code: 1

Command Output (stdout):
--
# RUN: at line 18
D:/github/actions-runner/_work/llvm/llvm/install/bin/clang++.exe  -Werror -I D:\github\actions-runner\_work\llvm\llvm\llvm\sycl\test-e2e\Matrix/Inputs  -fsycl -fsycl-targets=spir64  D:\github\actions-runner\_work\llvm\llvm\llvm\sycl\test-e2e\Matrix\SG32\joint_matrix_bf16_fill_k_cache_init.cpp -o D:\github\actions-runner\_work\llvm\llvm\build-e2e\Matrix\SG32\Output\joint_matrix_bf16_fill_k_cache_init.cpp.tmp.out -DINIT_LIST -DVNNI -ffp-model=precise
# executed command: D:/github/actions-runner/_work/llvm/llvm/install/bin/clang++.exe -Werror -I 'D:\github\actions-runner\_work\llvm\llvm\llvm\sycl\test-e2e\Matrix/Inputs' -fsycl -fsycl-targets=spir64 'D:\github\actions-runner\_work\llvm\llvm\llvm\sycl\test-e2e\Matrix\SG32\joint_matrix_bf16_fill_k_cache_init.cpp' -o 'D:\github\actions-runner\_work\llvm\llvm\build-e2e\Matrix\SG32\Output\joint_matrix_bf16_fill_k_cache_init.cpp.tmp.out' -DINIT_LIST -DVNNI -ffp-model=precise
# .---command stderr------------
# | In file included from D:\github\actions-runner\_work\llvm\llvm\llvm\sycl\test-e2e\Matrix\SG32\joint_matrix_bf16_fill_k_cache_init.cpp:27:
# | D:\github\actions-runner\_work\llvm\llvm\llvm\sycl\test-e2e\Matrix/Inputs\joint_matrix_bf16_fill_k_cache_impl.hpp:63:16: error: no member named 'high_resolution_clock' in namespace 'std::chrono'
# |    63 |   std::chrono::high_resolution_clock::time_point start =
# |       |   ~~~~~~~~~~~~~^
# | D:\github\actions-runner\_work\llvm\llvm\llvm\sycl\test-e2e\Matrix/Inputs\joint_matrix_bf16_fill_k_cache_impl.hpp:469:7: note: in instantiation of function template specialization 'test<sycl::ext::oneapi::bfloat16, float, 2ULL, 16ULL, 16ULL, 32ULL, 32ULL, 32ULL, 32ULL, 256ULL, 256ULL, 32ULL>' requested here
# |   469 |       test<bfloat16, float, VnniFactor, /*TM*/ 16, /*TN*/ 16, /*TK*/ 32,
# |       |       ^
# | D:\github\actions-runner\_work\llvm\llvm\llvm\sycl\test-e2e\Matrix/Inputs\joint_matrix_bf16_fill_k_cache_impl.hpp:64:20: error: no member named 'high_resolution_clock' in namespace 'std::chrono'
# |    64 |       std::chrono::high_resolution_clock::now();
# |       |       ~~~~~~~~~~~~~^
# | D:\github\actions-runner\_work\llvm\llvm\llvm\sycl\test-e2e\Matrix/Inputs\joint_matrix_bf16_fill_k_cache_impl.hpp:353:20: error: no member named 'high_resolution_clock' in namespace 'std::chrono'
# |   353 |       std::chrono::high_resolution_clock::now() - start;
# |       |       ~~~~~~~~~~~~~^
# | 3 errors generated.
# `-----------------------------
# error: command failed with exit status: 1

--
```
https://github.com/intel/llvm/actions/runs/13929259216/job/38985533716?pr=17335

### To reproduce

1. Include a code snippet that is as short as possible
2. Specify the command which should be used to compile the program
3. Specify the command which should be used to launch the program
4. Indicate what is wrong and what was expected


### Environment

- OS: [e.g Windows/Linux]
- Target device and vendor: [e.g. Intel GPU]
- DPC++ version: [e.g. commit hash or output of `clang++ --version`]
- Dependencies version: [e.g. the output of `sycl-ls --verbose`]


### Additional context

_No response_

--- a/sycl/test-e2e/Matrix/Inputs/joint_matrix_bf16_fill_k_cache_impl.hpp
+++ b/sycl/test-e2e/Matrix/Inputs/joint_matrix_bf16_fill_k_cache_impl.hpp
@@ -9,6 +9,10 @@
 #include <random>
 #include <sycl/usm.hpp>
 
+#ifdef _WIN32
+#include <chrono>
+#endif
+
 #ifdef SLM
 #include "slm_utils.hpp"
 #endif

